test/unit/helpers/rubygems_helper_test.rb

```diff
@@ -60,6 +60,7 @@ class RubygemsHelperTest < ActionView::TestCase
     version = build(:version)
     linkset = build(:linkset, docs: nil)
 
+    @virtual_path = "rubygems.show"
```

Comment:
Yeah I came to the same conclusion. This was just a stop-gap to get back to green. 

> On Nov 30, 2015, at 5:24 PM, Josef Šimánek <notifications@github.com> wrote:
> 
> In test/unit/helpers/rubygems_helper_test.rb:
> 
> > @@ -60,6 +60,7 @@ class RubygemsHelperTest < ActionView::TestCase
> >      version = build(:version)
> >      linkset = build(:linkset, docs: nil)
> >  
> > +    @virtual_path = "rubygems.show"
> I think that's because of view template key lookup is used in helpers..
> 
> link_to t(".links.rss") #=> (looks for @virtual_path + key.to_s)
> I think helper should not depend on @virtual_path. It will be better to use full key path for those string in those helpers and move them out of "view template" keys.
> 
> —
> Reply to this email directly or view it on GitHub.
>